### DISCUSSION TITLE
Preview in Editor

### DISCUSSION THREAD

**Discussion by stevenkodani-azra** [KyryloKuzyk/PrimeTween#47] [FAQ] [answered]
Hello!

We're in the process of transitioning our project from DOTween to PrimeTween. Our UI artist is used to using DOTweenAnimations to create and preview their work without needing to get engineering involved. I'm working on something comparable using TweenSettings. However, I can't figure out how to make PrimeTweens run in editor mode. I couldn't find anything in the docs regarding editor mode - is there a way to do this?

Thanks!

**Comment by KyryloKuzyk** [comment, marked as answer]
Hey,

Editor Preview is not yet supported. It's planned in PrimeTween Pro, but I can't share any estimates for now.

Unfortunately, there is currently no workflow for non-programmers. Using [TweenSettings](https://github.com/KyryloKuzyk/PrimeTween?tab=readme-ov-file#inspector-integration) requires basic C# knowledge to add an animation from code and connect it to a serialized TweenSettings field. And only after that this animation can be tweaked by non-programmers in the Inspector.

**Reply by stevenkodani-azra** [reply]
Awesome, thanks for confirming this for us. It's great to know that it's on your roadmap. We're looking forward to it!

**Reply by KyryloKuzyk** [reply]
Here is a simple alternative to the DOTweenAnimation component:
https://github.com/KyryloKuzyk/PrimeTween/discussions/36#discussioncomment-8090944

You can add more animation types to it, and your artists can use this component without the code. But it doesn’t work in the Editor, unfortunately.
